docs/vote-escrow/realms.mdx

```diff
@@ -0,0 +1,164 @@
+
+This guide outlines using Realms with the [Helium Wallet App](/wallets/helium-wallet-app). To use Realms on a desktop browser, a Solana-compatabile browser wallet can be used (such as Phantom or Backpack).
+
+The Helium Wallet App's browser can be accessed via the globe icon in the bottom navigation.
+
+<figure className="screensnippet-wrapper">
+    <img src={useBaseUrl('/img/realms/realmsbrowser.png')} />
+    <figcaption>veHNT can be managed via <code>https://realms.heliumvote.com/dao/hnt</code></figcaption>
+</figure>
+<br />
+
+Once your wallet has been connected to Realms, your token balances (or existing positions) will be displayed in the governance section of the Helium DAO.
+
+<figure className="screensnippet-wrapper">
+    <img src={useBaseUrl('/img/realms/availabletokens.png')} />
+    <figcaption>Stake positions can be made in the "My governance power" section of Realms</figcaption>
+</figure>
+<br />
+
+As long as HNT is available in your linked account, new stakes can be created by tapping on the "Lock Tokens" button within the governance view.
+
+<figure className="screensnippet-wrapper">
+    <img src={useBaseUrl('/img/realms/locktokensbtn.png')} />
+</figure>
+
+### Lockup Period
+In order to build a secure voting system, Helium only allows votes from token holders willing to align themselves with the longterm health of the Helium Network. This is done by allowing holders to receive veHNT in proportion with the amount of time that the tokens are staked for. 
+
+<figure className="screensnippet-wrapper">
+    <img src={useBaseUrl('/img/realms/lockup.png')} />
+    <figcaption>Selecting your lockup type, Cliff or Constant.</figcaption>
+</figure>
+
+#### Cliff
+Tokens are locked for a fixed duration and are released in full at the end of the defined lockup period.
+Vote weight declines linearly until release.
+
+:::note Example
+
+You lock 10.000 tokens for two years. They are then unavailable for the next two years. After this time, you can withdraw them again.
+
+:::
+
+#### Constant
+Tokens are locked indefinitely. At any time you can start the unlock process which lasts for the initially chosen lockup duration.
+Vote weight stays constant until you start the unlock process, then it declines linearly until release.
+
+
+:::note Example
+
+You lock 10.000 tokens with a lockup duration of one year. After two years you decide to start the unlocking process. Another year after that, you can withdraw the tokens.
+
+:::
+
+
+### Duration
+The duration of your cooldown is defined in days. The longer your tokens are staked, the greater the more veHNT will be issued. If you chose the 'Constant' lockup period, this duration takes affect after the 'Unlock' action is executed.
+
+While it is possible to set a higher cooldown than 5 years by using the 'Custom Duration', no additional vote weight multiplier will be added.
```

Comment:
The max lockup actually tops out at 4 years. The 5 years is a carry over from existing realms UI. Maybe we should just nix that?

Reply:
Yea if we can update the realms UI for this it'd be great. Will up date the copy here.